forest/cli/src/cli/wallet_cmd.rs

```diff
@@ -305,7 +285,7 @@ impl WalletCommands {
 /// in full accuracy for `ExactNotFixed` mode,
 /// mode to show data in SI units
 /// with 4 significant digits for `NotExactNotFixed` mode
-fn format_balance_string(
+pub fn format_balance_string(
```

Comment:
This function belongs in the utility crate together with `FormattingMode`

Reply:
moved